[PXCT-671] Wisgate Pro - Arduino Datasheet (#2426)
arduino/docs-content: content/hardware/05.pro-solutions/solutions-and-kits/wisgate-edge-pro/datasheet/datasheet.md

---
identifier: TPX00095-TPX00096-TPX00097
title: WisGate Edge Pro
type: pro
variant: 'Collective Datasheet'

> author: Julián Caro Linares, Taddy Ho Chung

Author: ```suggestion
author: Taddy Ho Chung, Julián Caro Linares
```

|-------------------------|------------------------------------------------------------------------------------------------------|
| Ethernet                | RJ45 (10/100BASE-T) port for network connectivity, status indicated by LED (Link/Activity)           |
| LoRa® Antennas          | External N-Type connectors (one for the 8-channel variant, two for the 16-channel variant)           |
| SIM Card Slot           | Nano SIM card slot for cellular connectivity (available on RAK7289CV2 cellular variant only)         |
| DC Input Connector      | DC jack for external power supply (12 VDC)                                                           |
| SD Card Slot            | MicroSD card (used internally for storing system logs and data)                                      |
| Reset Button            | Short press to restart, long press (≥5 sec) restores factory defaults                                |
| LED Indicators          | Indicate operational status for power, Ethernet, LoRa®, Wi-Fi® and optional cellular LTE connections |

<div style="background-color: #FFFFE0; border-left: 6px solid #FFD700; margin: 20px 0; padding: 15px;">
<p style="text-align: justify;">

> <strong>Warning:</strong> The microSD card in the SD card slot should not be ejected during operation, as doing so may affect the gateway's performance or lead to data loss. The card is reserved for system logs and operational data management.

Author: ```suggestion
<strong>Warning:</strong> The microSD card should not be ejected during operation, as doing so may affect the gateway's performance or lead to data loss. The card is reserved for system logs and operational data management.
```

Author: this way we do not have the jump problem

  - **Flickering:** Data transmission/reception activity.

**LED 3 (LoRa 1)**  
  - **ON:** LoRa channel 1 operational.
  - **OFF:** LoRa channel 1 inactive or disabled.
  - **Flickering:** LoRa channel 1 transmitting/receiving packets.

**LED 4 (WLAN): Access Point (AP) Mode:**
  - **ON:** Wi-Fi® Access Point is active.
  - **Flickering:** Data being transmitted or received in AP mode.

**LED 4 (WLAN): Station (STA) Mode:**
  - **Slow Flicker (1 Hz):** Wi-Fi® disconnected.
  - **ON:** Wi-Fi® connected.
  - **Flickering:** Data being transmitted or received in STA mode.

**LED 5 (LTE)** *(RAK7289CV2 cellular version only)*  
  - **Slow Flicker (1800 ms ON / 200 ms OFF):** Searching for cellular network.
  - **Slow Flicker (200 ms OFF / 1800 ms ON):** Cellular network idle (connected but no active data transfer).
  - **Fast Flicker (125 ms ON / 125 ms OFF):** Active data transmission/reception over cellular network.

**LED 6 (LoRa 2)** *(16-channel variant only)*  
  - **ON:** LoRa channel 2 operational.
  - **OFF:** LoRa channel 2 inactive or disabled.
  - **Flickering:** LoRa channel 2 transmitting/receiving packets.

#### RF Specifications

> The WisGate Edge Pro provides robust radio frequency performance, optimized for extended coverage and reliable communications:

Author: ```suggestion
The WisGate Edge Pro provides robust radio frequency performance, optimized for extended coverage and reliable communications.
```

#### Wi-Fi® Radio Specifications

> The gateway supports high performance Wi-Fi® connectivity with the following specifications:

Author: ```suggestion
The gateway supports high-performance Wi-Fi® connectivity with the following specifications:
```

| Operating Frequency            | ISM band: 2.412~2.472 GHz                                                                                                                                                                      |
| Operation Channels             | 2.4 GHz: Channels 1-13                                                                                                                                                                         |
| Antenna Configuration          | 2x2 MIMO (internal antenna)                                                                                                                                                                    |
| Transmit Power (Max)           | 802.11b: 19 dBm @1 Mbps, 19 dBm @11 Mbps <br> 802.11g: 18 dBm @6 Mbps, 16 dBm @54 Mbps <br> 802.11n (HT20): 18 dBm @MCS0, 16 dBm @MCS7 <br> 802.11n (HT40): 17 dBm @MCS0, 15 dBm @MCS7         |
| Receiver Sensitivity (Typical) | 802.11b: −95 dBm @1 Mbps, −88 dBm @11 Mbps <br> 802.11g: −90 dBm @6 Mbps, −75 dBm @54 Mbps <br> 802.11n (HT20): −89 dBm @MCS0, −72 dBm @MCS7 <br> 802.11n (HT40): −86 dBm @MCS0, −68 dBm @MCS7 |

#### LoRa® Radio Specifications

> The LoRa® functionality is provided by **SX1303** (RAK5146) based concentrator modules, offering great performance for long range communication. The gateway can support either 8 or optionally 16 LoRa channels, allowing flexible scalability.

Author: ```suggestion
The LoRa® functionality is provided by **SX1303** (RAK5146) based concentrator modules, offering great performance for long range communication. The gateway can support either 8x or optionally 16x LoRa channels, allowing flexible scalability.
```

| **Specification**          | **Details**                                                    |
|----------------------------|----------------------------------------------------------------|

> | LoRa® Module               | SX1303 (RAK5146) mPCIe (up to 2 modules)                       |

Author: ```suggestion
| LoRa® Module               | SX1303 (RAK5146) mPCIe (up to 2x modules)                       |
```

> | Channel Configuration      | 8 channels standard, 16 channels optional                      |

Author: ```suggestion
| Channel Configuration      | 8x channels standard, 16x channels optional                      |
```

</p>
</div>

### Power Specification

<p style="text-align: justify;">
The WisGate Edge Pro gateway supports multiple power supply options suitable for various installation scenarios:
</p>

- **Power over Ethernet (PoE)**: Supports IEEE 802.3af standard (37 to 57 VDC input voltage). Use an IEEE 802.3af compliant PoE injector or switch for proper operation. The PoE port provides 10/100 Mbps Ethernet connectivity.
- **External DC Power Supply**: Supports a dedicated external 12 VDC power source via the DC barrel jack. Use the 12V/1A power adapter or a regulated 12 VDC supply with at least 2 A capacity to ensure stable operation.

| **Property**                | **Min** | **Typical** | **Max** | **Unit** |
|-----------------------------|---------|-------------|---------|----------|
| PoE Input Voltage (802.3af) | 37      | -           | 57      | VDC      |
| DC Input Voltage            | 11      | 12          | 13      | VDC      |
| Power Consumption           | -       | 12          | 17      | W        |

<div style="background-color: #FFFFE0; border-left: 6px solid #FFD700; margin: 20px 0; padding: 15px;">
<p style="text-align: justify;">

> <strong>Warning:</strong> Always use IEEE 802.3af compliant PoE injectors or switches to prevent potential damage or malfunction. If using a DC power supply, ensure it is a regulated 12V source capable of supplying at least 2A to accommodate peak power consumption.

Author: ```suggestion
<strong>Warning:</strong> Always use IEEE 802.3af compliant PoE injectors or switches to prevent potential damage or malfunction. If using a DC power supply, ensure it is a regulated 12 VDC source capable of supplying at least 2 A to accommodate peak power consumption.
```

<p style="text-align: justify;">
<strong>Note:</strong> Current consumption may vary depending on the operational load (LoRa® traffic, Wi-Fi®, LTE usage), environmental conditions and peripheral device connections. If operating at maximum load, use DC power instead of PoE, as 802.3af PoE may not supply sufficient power under peak conditions.
</p>
</div>

## Device Operation

<p style="text-align: justify;">
The WisGate Edge Pro gateway is designed for professional IoT environments requiring reliable LoRa®, Wi-Fi® and cellular connectivity. The gateway provides easy setup and deployment, supporting various connectivity options including Ethernet, Wi-Fi and cellular LTE (with an optional module). The device includes dedicated interfaces for ease of configuration, operation and monitoring.
</p>

The WisGate Edge Pro gateway can easily be configured and managed through a built-in web interface accessible via Ethernet or Wi-Fi®. For step-by-step instructions, please refer to [the WisGate Edge Pro Getting Started documentation](https://docs.arduino.cc/tutorials/wisgate-edge-pro/getting-started/).

### Online Resources

> Now that you have gone through the basics of what you can do with the board, you can explore the endless possibilities it provides by checking exciting projects on ProjectHub **[4]**, the Arduino Library Reference **[5]**, and the online store **[6]**; where you will be able to complement your board with sensors, actuators and more.

Author: ```suggestion
Now that you have gone through the basics of what you can do with the board, you can explore the endless possibilities it provides by checking exciting projects on ProjectHub **[1]**, the Arduino Library Reference **[2]**, and the online store **[3]**; where you will be able to complement your board with sensors, actuators and more.
```

### Device Recovery

In the event the WisGate Edge Pro gateway becomes unresponsive or requires resetting to factory defaults:

- **Short press** the reset button to restart the gateway (soft reset).

> - **Long press** (≥5 seconds): Restores gateway to factory default settings, erasing user configured parameters and resetting network settings. Use this option cautiously, as all user configurations and data will be permanently lost.

Author: ```suggestion
- **Long press** (≥ 5 seconds): Restores gateway to factory default settings, erasing user configured parameters and resetting network settings. Use this option cautiously, as all user configurations and data will be permanently lost.
```

### Declaration of Conformity to EU RoHS & REACH 211 01/19/2021

Arduino boards are in compliance with RoHS 2 Directive 2011/65/EU of the European Parliament and RoHS 3 Directive 2015/863/EU of the Council of 4 June 2015 on the restriction of the use of certain hazardous substances in electrical and electronic equipment.

| **Substance**                          | **Maximum Limit (ppm)** |
|----------------------------------------|-------------------------|
| Lead (Pb)                              | 1000                    |
| Cadmium (Cd)                           | 100                     |
| Mercury (Hg)                           | 1000                    |
| Hexavalent Chromium (Cr6+)             | 1000                    |
| Poly Brominated Biphenyls (PBB)        | 1000                    |
| Poly Brominated Diphenyl ethers (PBDE) | 1000                    |
| Bis(2-Ethylhexyl) phthalate (DEHP)     | 1000                    |
| Benzyl butyl phthalate (BBP)           | 1000                    |
| Dibutyl phthalate (DBP)                | 1000                    |
| Diisobutyl phthalate (DIBP)            | 1000                    |

> <div style="page-break-after: always;"></div>

Author: ```suggestion
```

## Company Information

<!---
Company information: Make sure is updated and always the same.
-->

| **Company name** |              **Arduino S.r.l.**              |
| :--------------: | :------------------------------------------: |
| Company address  | Via Andrea Appiani, 25 - 20900 Monza (Italy) |

## Reference Documentation

|        **Ref**        | **Link**                                                                 |
|:---------------------:|--------------------------------------------------------------------------|
|      Project Hub      | https://create.arduino.cc/projecthub?by=part&part_id=11332&sort=trending |
|   Library Reference   | https://www.arduino.cc/reference/en/                                     |
|     Online Store      | https://store.arduino.cc/                                                |
| WisGate Edge Pro Docs | https://docs.arduino.cc/hardware/wisgate-edge-pro/                       |

## Revision History

|  **Date**  | **Revision** |                      **Changes**                       |
| :--------: | :----------: | :----------------------------------------------------: |

> | 12/03/2025 |      1       |                     First release                      |

Author: ```suggestion
| 14/03/2025 |      2       |                     Format improvements       |
| 12/03/2025 |      1       |                     First release                      |
```
